applications/serial_lte_modem/overlay-native_tls.conf

```diff
@@ -44,6 +44,8 @@ CONFIG_MBEDTLS_ECDSA_C=y
 CONFIG_PSA_WANT_ALG_ECDSA=y
 CONFIG_MBEDTLS_ECDSA_DETERMINISTIC=y
 CONFIG_PSA_WANT_ALG_DETERMINISTIC_ECDSA=y
+CONFIG_PSA_WANT_KEY_TYPE_ECC_KEY_PAIR=y # dep for DETERMINISTIC_ECDSA
+CONFIG_PSA_WANT_ALG_HMAC=y # dep for DETERMINISTIC_ECDSA
```

Comment:
Will conflict with https://github.com/nrfconnect/sdk-nrf/pull/13729 as they do the same changes.